vendor/jasmine-only/jasmine-only.js

```diff
@@ -0,0 +1,98 @@
+/* jasmine-only - 0.1.1
+ * Exclusivity spec helpers for jasmine: `describe.only` and `it.only`
+ * https://github.com/davemo/jasmine-only
+ */
+(function() {
```

Comment:
Would love a PR to the jasmine-only repo that achieved this if you think it's worthwhile :)